src/test_l1_hdmi_cec_driver.c

```diff
@@ -1342,7 +1246,7 @@ void test_hdmicec_hal_l1_setTxCallback_positive( void )
  */
 void test_hdmicec_hal_l1_hdmiCecTx_sinkDevice_negative( void )
 {
-    int result=HDMI_CEC_IO_SENT_AND_ACKD;
+    HDMI_CEC_STATUS result =HDMI_CEC_IO_SENT_AND_ACKD;
```

Comment:
in the below lines, where ever we have /*invalid inputs */, can we specify what is the invalid param here?

Reply:
Addressed the review